scanreader/scans.py

```diff
@@ -671,6 +672,114 @@ class Scan2020(Scan5Point3):
     """ ScanImage 2020"""
     pass
 
+class Scan2021(Scan5Point3):
+    """ ScanImage 2021"""
+
+    # New meta data that gets saved into the header when Mini2p unwarping
+    # toolbox was used to correct the data 
+    # 
+    # SI.Custom.MINI2P.MINI2P_Corrected | 'true' or '1' 
+    # SI.Custom.MINI2P.tf_path | str | transformMatrixDirectory
+    # SI.Custom.MINI2P.tfused | str | TransformMatrix_used
+    # SI.Custom.MINI2P.MINI2P_system | str | 
+    # SI.Custom.MINI2P.MINI2P_scope | str |
+    # SI.Custom.MINI2P.MINI2P_objective | str | 
+
+    # Similar entries will be created on the level of the MDF 
+    # transformMatrixDirectory
+    # system
+    # scope
+    # objective
+
+    # @property
+    # @MoserValidation(validated=True)
+    # def is_distortion_corrected(self):
+    #     match = re.search(r"SI.*.MINI2P_Corrected\s*?=\s*?(?P<is_corrected>.*)", string)
+    #     is_corrected = match.group("is_corrected").strip() in ("true", "1")
+    #     return is_corrected
+    @property
+    @MoserValidation(validated=True)
+    def transform_matrix_path(self):
+        ''' 
+        Path of transform matrix object (.mat file)
+        that was used for unwarping (correcting) raw scanimage data
+        '''
+        match = re.search(r'SI.*.tf_path\s*?=\s*?(?P<tf_path>.*)', self.header)
+        if match:
+            path = matlabstr2py(match.group('tf_path').strip())
+            return path
+        else:
+            return None
+
+    @property
+    @MoserValidation(validated=True)
+    def transform_matrix_index(self):
```

Comment:
Same here, maybe `MINI2P_transform_matrix_index`

I would also use "WARNING" rather than "CAVE" - "cave" isn't wrong, but it's somewhat more obscure, at least in english

Reply:
Done